src/components/MemberArea/model.js

```diff
@@ -17,6 +17,8 @@ const linkedinValidation = value => !value || linkedinPattern.test(value);
 const facebookValidation = value => !value || facebookPattern.test(value);
 const twitterValidation = value => !value || twitterPattern.test(value);
 const githubValidation = value => !value || githubPattern.test(value);
+const nameValidation = value =>
+  value.length > 3 && value.length <= 50 && /^\S+(\s\S+)+$/.test(value);
```

Comment:
I am not sure if the backend strips whitespace in the beginning and the end so I added that in the regex here.